### ISSUE TITLE
Sheet close button size not accessible for touch

### ISSUE THREAD

**Issue by kof** [shadcn-ui/ui#1333]
Minimal  target size is 44px according to WCAG, currently that button is 16px

<img width="384" alt="Screenshot 2023-08-24 at 16 26 16" src="https://github.com/shadcn-ui/ui/assets/52824/0a53686e-1dc2-4781-9a28-8c99ce789f7c">

**Comment by shadcn** [comment]
This issue has been automatically closed because it received no activity for a while. If you think it was closed by accident, please leave a comment. Thank you.

**Comment by kof** [comment]
still unsolved
